includes/api/class-wc-admin-rest-products-controller.php

```diff
@@ -145,14 +171,14 @@ public static function add_wp_query_product_search_join( $join, $wp_query ) {
 	 * @param object $wp_query WP_Query object.
 	 * @return string
 	 */
-	public function add_wp_query_product_search_group_by( $groupby, $wp_query ) {
+	public function add_wp_query_group_by( $groupby, $wp_query ) {
 		global $wpdb;
 
-		$search = trim( $wp_query->get( 'search' ) );
-		if ( $search && empty( $groupby ) ) {
+		$search       = trim( $wp_query->get( 'search' ) );
+		$low_in_stock = trim( $wp_query->get( 'low_in_stock' ) );
```

Comment:
Why `trim()` here? If `prepare_objects_query()` handled coercion, would this be needed?

Reply:
You're right, not necessary.